utils.js

```diff
@@ -1,22 +1,25 @@
-const fs = require('fs-extra');
```

Comment:
Similar to the comment made on ./combine.js, would it be better to use const for the variables here?

Reply:
Made the changes